<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Workouts</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QFrame" name="summframe">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTool_bar"/>
    <addaction name="actionStatus_bar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddWorkout"/>
    <addaction name="actionDeleteWorkout"/>
    <addaction name="actionSubmit"/>
    <addaction name="actionRevert"/>
    <addaction name="separator"/>
    <addaction name="actionWorkoutType"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>resources/exit.svg</normaloff>resources/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionTool_bar">
   <property name="text">
    <string>Tool bar</string>
   </property>
  </action>
  <action name="actionStatus_bar">
   <property name="text">
    <string>Status bar</string>
   </property>
  </action>
  <action name="actionAddWorkout">
   <property name="icon">
    <iconset>
     <normaloff>resources/add.svg</normaloff>resources/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add workout</string>
   </property>
  </action>
  <action name="actionDeleteWorkout">
   <property name="icon">
    <iconset>
     <normaloff>resources/delete.svg</normaloff>resources/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete workout</string>
   </property>
  </action>
  <action name="actionSubmit">
   <property name="icon">
    <iconset>
     <normaloff>resources/submit.svg</normaloff>resources/submit.svg</iconset>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="icon">
    <iconset>
     <normaloff>resources/revert.svg</normaloff>resources/revert.svg</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionWorkoutType">
   <property name="text">
    <string>Workout type</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>resources/settings.svg</normaloff>resources/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
